11-database-connections.Rmd

```diff
@@ -0,0 +1,288 @@
+---
+title: "Database connections in R"
+subtitle: "Session 11"
+institute: "NHS-R Community"
+output:
+  xaringan::moon_reader:
+    css: 
+      - default
+      - css/nhsr.css
+      - css/nhsr-fonts.css
+    lib_dir: libs
+    seal: false
+    self-contained: true
+    nature:
+      highlightStyle: googlecode
+      highlightLines: true
+      highlightLanguage: ["r"]
+      countIncrementalSlides: false
+      ratio: "16:9"
+    includes:
+      after_body: [css/insert-logo.html]
+
+---
+
+
+
+```{r libs, include=FALSE}
+
+knitr::opts_chunk$set(echo = FALSE, eval = FALSE, message = FALSE, warning = FALSE)
+
+#Runs packages if available
+library(knitr)
+library(tidyverse)
+library(xaringan)
+library(nhsrtheme)
+library(magick)
```

Comment:
add these to the Suggests in DESCRIPTION (Magick, dbplyr,  + any others I have missed)